doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -60,56 +66,55 @@ Communication between Python and the AEDT API were translated through an interme
 
 `gRPC <https://grpc.io/>`_ is a modern open source high performance Remote Procedure Call (RPC)
 framework that can run in any environment and supports client/server remote calls.
-Starting from 2022R2 and later, AEDT API has replaced COM interface with gRPC interface.
+Starting from 2022R2 the AEDT API has replaced the COM interface with a gRPC interface.
 
 
-.. list-table:: gRPC Compatibility Table
-   :widths: 25 25 75 75
+.. list-table:: *gRPC Compatibility:*
+   :widths: 65 65 65
    :header-rows: 1
 
-   * -
-     - <2022R2
+   * - < 2022R2
      - 2022R2
-     - >2022R2
-   * - AEDT
-     - Only ``Python.NET``
-     - gRPC Beta
-     - gRPC Available
-   * - AEDT
-     - Only ``Python.NET``
-     - ``Python.NET`` default.
-       gRPC: pyaedt.settings.use_grpc_api = True
-     - gRPC default
-       ``Python.NET``: pyaedt.settings.use_grpc_api = False
+     - > 2022R2
+   * - Only Python.NET
```

Comment:
Add Python.NET to the accept.list file here: C:\AnsysDev\GitRepos\PyAnsys\pyaedt\doc\styles\Vocab\ANSYS

Reply:
will do. Thanks!